[2019-04-24 13:27:14 +0000] [10] [INFO] Starting gunicorn 19.9.0
[2019-04-24 13:27:14 +0000] [10] [INFO] Listening at: http://127.0.0.1:8000 (10)
[2019-04-24 13:27:14 +0000] [10] [INFO] Using worker: sync
[2019-04-24 13:27:14 +0000] [32] [INFO] Booting worker with pid: 32
[2019-04-24 13:27:14 +0000] [34] [INFO] Booting worker with pid: 34
[2019-04-24 13:27:14 +0000] [35] [INFO] Booting worker with pid: 35
[2019-04-24 13:27:14 +0000] [36] [INFO] Booting worker with pid: 36
/opt/graphite/webapp/graphite/local_settings.py:334: UserWarning: SECRET_KEY is set to an unsafe default. This should be set in local_settings.py for better security
/opt/graphite/webapp/graphite/local_settings.py:334: UserWarning: SECRET_KEY is set to an unsafe default. This should be set in local_settings.py for better security
/opt/graphite/webapp/graphite/local_settings.py:334: UserWarning: SECRET_KEY is set to an unsafe default. This should be set in local_settings.py for better security
/opt/graphite/webapp/graphite/local_settings.py:334: UserWarning: SECRET_KEY is set to an unsafe default. This should be set in local_settings.py for better security
/usr/lib/python2.7/site-packages/whitenoise/base.py:104: UserWarning: No directory at: /opt/graphite/static/
  warnings.warn(u'No directory at: {}'.format(root))
/usr/lib/python2.7/site-packages/whitenoise/base.py:104: UserWarning: No directory at: /opt/graphite/static/
  warnings.warn(u'No directory at: {}'.format(root))
/usr/lib/python2.7/site-packages/whitenoise/base.py:104: UserWarning: No directory at: /opt/graphite/static/
  warnings.warn(u'No directory at: {}'.format(root))
/usr/lib/python2.7/site-packages/whitenoise/base.py:104: UserWarning: No directory at: /opt/graphite/static/
  warnings.warn(u'No directory at: {}'.format(root))
[2019-04-24 13:27:58 +0000] [10] [INFO] Handling signal: term
[2019-04-24 13:27:58 +0000] [34] [INFO] Worker exiting (pid: 34)
[2019-04-24 13:27:58 +0000] [35] [INFO] Worker exiting (pid: 35)
[2019-04-24 13:27:58 +0000] [36] [INFO] Worker exiting (pid: 36)
[2019-04-24 13:27:58 +0000] [32] [INFO] Worker exiting (pid: 32)
[2019-04-24 13:27:58 +0000] [10] [INFO] Shutting down: Master
[2019-04-24 13:31:45 +0000] [24] [INFO] Starting gunicorn 19.9.0
[2019-04-24 13:31:45 +0000] [24] [INFO] Listening at: http://127.0.0.1:8000 (24)
[2019-04-24 13:31:45 +0000] [24] [INFO] Using worker: sync
[2019-04-24 13:31:45 +0000] [37] [INFO] Booting worker with pid: 37
[2019-04-24 13:31:45 +0000] [40] [INFO] Booting worker with pid: 40
[2019-04-24 13:31:45 +0000] [41] [INFO] Booting worker with pid: 41
/opt/graphite/webapp/graphite/local_settings.py:334: UserWarning: SECRET_KEY is set to an unsafe default. This should be set in local_settings.py for better security
/usr/lib/python2.7/site-packages/whitenoise/base.py:104: UserWarning: No directory at: /opt/graphite/static/
  warnings.warn(u'No directory at: {}'.format(root))
[2019-04-24 13:31:45 +0000] [42] [INFO] Booting worker with pid: 42
/opt/graphite/webapp/graphite/local_settings.py:334: UserWarning: SECRET_KEY is set to an unsafe default. This should be set in local_settings.py for better security
/opt/graphite/webapp/graphite/local_settings.py:334: UserWarning: SECRET_KEY is set to an unsafe default. This should be set in local_settings.py for better security
/usr/lib/python2.7/site-packages/whitenoise/base.py:104: UserWarning: No directory at: /opt/graphite/static/
  warnings.warn(u'No directory at: {}'.format(root))
/usr/lib/python2.7/site-packages/whitenoise/base.py:104: UserWarning: No directory at: /opt/graphite/static/
  warnings.warn(u'No directory at: {}'.format(root))
/opt/graphite/webapp/graphite/local_settings.py:334: UserWarning: SECRET_KEY is set to an unsafe default. This should be set in local_settings.py for better security
/usr/lib/python2.7/site-packages/whitenoise/base.py:104: UserWarning: No directory at: /opt/graphite/static/
  warnings.warn(u'No directory at: {}'.format(root))
[2019-04-24 13:32:08 +0000] [24] [INFO] Handling signal: term
[2019-04-24 13:32:08 +0000] [41] [INFO] Worker exiting (pid: 41)
[2019-04-24 13:32:08 +0000] [40] [INFO] Worker exiting (pid: 40)
[2019-04-24 13:32:08 +0000] [37] [INFO] Worker exiting (pid: 37)
[2019-04-24 13:32:08 +0000] [42] [INFO] Worker exiting (pid: 42)
[2019-04-24 13:32:08 +0000] [24] [INFO] Shutting down: Master
[2019-04-25 14:28:10 +0000] [16] [INFO] Starting gunicorn 19.9.0
[2019-04-25 14:28:10 +0000] [16] [INFO] Listening at: http://127.0.0.1:8000 (16)
[2019-04-25 14:28:10 +0000] [16] [INFO] Using worker: sync
[2019-04-25 14:28:10 +0000] [38] [INFO] Booting worker with pid: 38
[2019-04-25 14:28:10 +0000] [41] [INFO] Booting worker with pid: 41
/opt/graphite/webapp/graphite/local_settings.py:334: UserWarning: SECRET_KEY is set to an unsafe default. This should be set in local_settings.py for better security
[2019-04-25 14:28:10 +0000] [42] [INFO] Booting worker with pid: 42
/opt/graphite/webapp/graphite/local_settings.py:334: UserWarning: SECRET_KEY is set to an unsafe default. This should be set in local_settings.py for better security
/usr/lib/python2.7/site-packages/whitenoise/base.py:104: UserWarning: No directory at: /opt/graphite/static/
  warnings.warn(u'No directory at: {}'.format(root))
/usr/lib/python2.7/site-packages/whitenoise/base.py:104: UserWarning: No directory at: /opt/graphite/static/
  warnings.warn(u'No directory at: {}'.format(root))
/opt/graphite/webapp/graphite/local_settings.py:334: UserWarning: SECRET_KEY is set to an unsafe default. This should be set in local_settings.py for better security
[2019-04-25 14:28:10 +0000] [43] [INFO] Booting worker with pid: 43
/usr/lib/python2.7/site-packages/whitenoise/base.py:104: UserWarning: No directory at: /opt/graphite/static/
  warnings.warn(u'No directory at: {}'.format(root))
/opt/graphite/webapp/graphite/local_settings.py:334: UserWarning: SECRET_KEY is set to an unsafe default. This should be set in local_settings.py for better security
/usr/lib/python2.7/site-packages/whitenoise/base.py:104: UserWarning: No directory at: /opt/graphite/static/
  warnings.warn(u'No directory at: {}'.format(root))
[2019-04-25 14:29:01 +0000] [16] [INFO] Handling signal: term
[2019-04-25 14:29:01 +0000] [41] [INFO] Worker exiting (pid: 41)
[2019-04-25 14:29:01 +0000] [42] [INFO] Worker exiting (pid: 42)
[2019-04-25 14:29:01 +0000] [43] [INFO] Worker exiting (pid: 43)
[2019-04-25 14:29:01 +0000] [38] [INFO] Worker exiting (pid: 38)
[2019-04-25 14:29:01 +0000] [16] [INFO] Shutting down: Master
[2019-04-25 14:31:59 +0000] [16] [INFO] Starting gunicorn 19.9.0
[2019-04-25 14:31:59 +0000] [16] [INFO] Listening at: http://127.0.0.1:8000 (16)
[2019-04-25 14:31:59 +0000] [16] [INFO] Using worker: sync
[2019-04-25 14:31:59 +0000] [35] [INFO] Booting worker with pid: 35
[2019-04-25 14:31:59 +0000] [38] [INFO] Booting worker with pid: 38
/opt/graphite/webapp/graphite/local_settings.py:334: UserWarning: SECRET_KEY is set to an unsafe default. This should be set in local_settings.py for better security
[2019-04-25 14:31:59 +0000] [39] [INFO] Booting worker with pid: 39
/usr/lib/python2.7/site-packages/whitenoise/base.py:104: UserWarning: No directory at: /opt/graphite/static/
  warnings.warn(u'No directory at: {}'.format(root))
/opt/graphite/webapp/graphite/local_settings.py:334: UserWarning: SECRET_KEY is set to an unsafe default. This should be set in local_settings.py for better security
[2019-04-25 14:31:59 +0000] [40] [INFO] Booting worker with pid: 40
/usr/lib/python2.7/site-packages/whitenoise/base.py:104: UserWarning: No directory at: /opt/graphite/static/
  warnings.warn(u'No directory at: {}'.format(root))
/opt/graphite/webapp/graphite/local_settings.py:334: UserWarning: SECRET_KEY is set to an unsafe default. This should be set in local_settings.py for better security
/usr/lib/python2.7/site-packages/whitenoise/base.py:104: UserWarning: No directory at: /opt/graphite/static/
  warnings.warn(u'No directory at: {}'.format(root))
/opt/graphite/webapp/graphite/local_settings.py:334: UserWarning: SECRET_KEY is set to an unsafe default. This should be set in local_settings.py for better security
/usr/lib/python2.7/site-packages/whitenoise/base.py:104: UserWarning: No directory at: /opt/graphite/static/
  warnings.warn(u'No directory at: {}'.format(root))
